plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -120,6 +120,9 @@ public abstract class AbstractTrinoCatalog
         implements TrinoCatalog
 {
     public static final String TRINO_CREATED_BY_VALUE = "Trino Iceberg connector";
+    public static final String ICEBERG_VIEW_SQL_DIALECT = "trino";
+    public static final String ICEBERG_VIEW_RUN_AS_OWNER = "trino_run_as_owner";
```

Comment:
Should we name the property `trino.run_as_owner`?

Reply:
I think that would be better since typically we namespace properties and separate those with a period. Right now, this is the only property, but down the line there can be more in Trino and we'd want to follow the established pattern.

I think we also want to remove the snake case in favor of dashes. That is the pattern that iceberg typically follows for property names. see https://iceberg.apache.org/docs/1.5.0/view-configuration/?h=replace.dro#view-properties

So the final property name would be:

```
trino.run-as-owner
```

Reply:
Sounds good to me